src/assets/src/validation.ts

```diff
@@ -66,7 +66,8 @@ export const queueLocationSchema = string().defined().trim().max(100, createRema
 export const uniqnameSchema = string().defined().trim().lowercase()
     .min(3, 'Uniqnames must be at least 3 characters long.')
     .max(8, 'Uniqnames must be at most 8 characters long.')
-    .matches(/^[a-z]+$/i, 'Uniqnames cannot contain non-alphabetical characters.');
+    .matches(/^[^@]*$/, "You need to use a uniqname and not an email address")
```

Comment:
Please rephrase this to "Please use your uniqname instead of your email address."

Reply:
Just updated the message!